### PR TITLE
feat: Option to see Website

### LINKED ISSUE #1268: Add Option to visit website from app
**Describe the chore**
An option should be there to see the website.

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #1268 

Changes: Now website can be seen.

Screenshots for the change:
Screenshot_20190311-072301

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -166,6 +166,8 @@
     <string name="title_terms_of_service">Terms of Service</string>
     <string name="key_terms_of_service" translatable="false">terms_of_service</string>
     <string name="legal">Legal</string>
+    <string name="visit_website">Visit Website</string>
+    <string name="key_visit_website">Visit Website</string>
```
Comment (ShridharGoel) [inline app/src/main/res/values/strings.xml]: Change the value to visit_website

File: app/src/main/res/xml/settings.xml [hunk 2]
```diff
@@ -2,64 +2,69 @@
...
-            android:title="@string/title_terms_of_service" />
+            android:title="@string/title_terms_of_service"
+            app:iconSpaceReserved="false" />
 
         <Preference
-            app:iconSpaceReserved="false"
             android:key="@string/key_cookie_policy"
-            android:title="@string/cookie_policy" />
+            android:title="@string/cookie_policy"
+            app:iconSpaceReserved="false" />
```
Comment (ShridharGoel) [inline app/src/main/res/xml/settings.xml]: Many unrelated changes in this file

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (somenath1435) [reply app/src/main/res/values/strings.xml]: @ShridharGoel can you be more specific?
First string is for text and second for it's key

File: app/src/main/res/xml/settings.xml (hunk 2 above)
Comment (somenath1435) [reply app/src/main/res/xml/settings.xml]: > this
It's android studio autoformatted.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/res/values/strings.xml]: Yes so just change the value of the second string to 'visit_website' instead of 'Visit Website'

**Comment by iamareebjamal** [comment]
Why was the previous PR closed. I told you about the fossasia guidelines about PRs

**Comment by somenath1435** [comment]
@iamareebjamal there were too many problems regarding merge conflicts

**Comment by iamareebjamal** [comment]
Then learn to solve merge conflicts. I understand that you are planning on applying for GSOC. About 4-5 people work more frequently in GSOC period. Are you going to close PRs if there's a merge conflict?

**Comment by somenath1435** [comment]
please see if any other changes are required.

**Comment by somenath1435** [comment]
@iamareebjamal I think this PR is ready.
